spark/datasources-v2/src/main/java/com/alibaba/graphar/GeneralParams.java

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
It seems that the GeneralParams could be the common part of GraphAr-spark, does we need to duplicated it in datasource?

Reply:
They are used inside data sources, and I duplicate it to avoid circular dependency (common depends of datasources because of Neo4j and if datasources will depend of common we will have a circular dependency)

Reply:
ok